jsapp/js/components/formLanding.es6

```diff
@@ -181,16 +181,156 @@ var FormLanding = React.createClass({
             })}
           </bem.FormView__group>
         </bem.FormView__cell>
-        {this.state.deployed_versions.length > 3 &&
-          <bem.FormView__cell m={['centered', 'padding']}>
-            <button className="mdl-button mdl-js-button mdl-button--colored" onClick={this.toggleDeploymentHistory}>
+        {this.state.deployed_versions.length > 1 &&
+          <bem.FormView__cell m={['centered']}>
+            <button className="mdl-button mdl-button--colored" onClick={this.toggleDeploymentHistory}>
               {this.state.historyExpanded ? t('Hide full history') : t('Show full history')}
             </button>
           </bem.FormView__cell>
         }
       </bem.FormView__row>
       );
   },
+  renderCollectData () {
+    var deployment__links = this.state.deployment__links;
+
+    var available__links = {
```

Comment:
The order of these `available__links` is not necessarily preserved. The best way to do this would be a `es6-map` which is equivalent to python's `OrderedDict`